lib/danger/request_source/bitbucket_server_api.rb

```diff
@@ -0,0 +1,70 @@
+# coding: utf-8
+require "danger/helpers/comments_helper"
+
+module Danger
+  module RequestSources
+    class BitbucketServerAPI
```

Comment:
To me this is just a supporting class for the request source. This class doesn't inherit from the base request source so it doesn't seem right for it to live next to the other request sources. I would put it in `lib/danger/api_clients` or something like that